java/pom.xml

```diff
@@ -925,6 +939,16 @@
                 <artifactId>okio-jvm</artifactId>
                 <version>${okio.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.roaringbitmap</groupId>
+                <artifactId>RoaringBitmap</artifactId>
+                <version>${roaringbitmap.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.roaringbitmap</groupId>
+                <artifactId>shims</artifactId>
+                <version>${shims.version}</version>
```

Comment:
It looks like this dependency isn't actually used after upgrading RoaringBitmap.